### ISSUE TITLE
 kittiviewer cannot show the detection result when I evaluate testing data in kitti.

### ISSUE THREAD

**Issue by duaaany** [traveller59/second.pytorch#435]
I have trained the model and I want to evaluate the testing data in kitti dataset.

I modified the car.fhd.config:  
eval_input_reader: {
  dataset: {
    dataset_class_name: "KittiDataset"
    # kitti_info_path: "/root/data/new/second.pytorch/second/KITTI_DATASET_ROOT/kitti_infos_val.pkl"
    kitti_info_path: "/root/data/new/second.pytorch/second/KITTI_DATASET_ROOT/kitti_infos_test.pkl"
    kitti_root_path: "/root/data/new/second.pytorch/second/KITTI_DATASET_ROOT"
  }

And then use the command: " python ./pytorch/train.py evaluate --config_path=./configs/car.fhd.config --model_dir=model_dir/evalmodel1 --measure_time=True --batch_size=1 "

it outputs:
[  41 1600 1408]
feature_map_size [1, 200, 176]
remain number of infos: 7518
Generate output labels...
[100.0%][===================>][18.55it/s][06:48>00:00]     
generate label finished(18.35/s). start eval:
avg example to torch time: 1.279 ms
avg prep time: 4.002 ms
avg voxel_feature_extractor time = 0.171 ms
avg middle forward time = 27.615 ms
avg rpn forward time = 6.117 ms
avg predict time = 14.980 ms

model folder only has one step result.
Evaluation succeed or not?

When I try to see the visualization result in kittiviewer, it doesn't have the detection box, what should I do to get the visualization results for evaluation testing data?

Thank you!
![image](https://user-images.githubusercontent.com/43082781/104163836-02fa7b00-5432-11eb-8615-9e5fb0a617bc.png)

![image](https://user-images.githubusercontent.com/43082781/104163724-d47ca000-5431-11eb-8ad5-479672f6e05f.png)

**Comment by lyy15871794839** [comment]
I have the same problom,do you solve it?

**Comment by wj-shin** [comment]
do you solve it?

**Comment by urbansound8K** [comment]
I went through it and solved it by making sure that all the paths of the generated files are correct.
